job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -372,14 +372,8 @@ definitions:
         description: The time the job status or progress last changed
       status:
         type: string
-        description: The status of the job
-        enum:
-          - Active
-          - Cancelled
-          - Completed
-          - Failed
-          - Paused
-          - Waiting
+        string:
```

Comment:
What is `string:`?  How is it different to `type: string` which seems to make much more sense?